start.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/python2.7 -u
```

Comment:
start.py does have mode 775 (`chmod +x`), and can currently be executed directly by the user -- in fact, our README.md currently suggests running `docker-compose run --use-aliases -e FAKE_DNS=172.17.0.1 --service-ports boulder ./start.py` in order to spin up all the services and be able to interact with and debug them without running the tests. For that to work, it does need a shebang line of some kind, even if that line should be updated to python3.

Reply:
Really great catch here. Thanks so much for the thorough review.